packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.tsx

```diff
@@ -37,13 +39,15 @@ export class AtomicResultIcon {
   }
 
   public render() {
-    const icon = this.icon;
-    return icon ? (
-      <atomic-icon icon={'assets://' + icon} class={icon}></atomic-icon>
-    ) : (
-      <slot>
-        <atomic-icon icon="assets://custom" class="custom"></atomic-icon>
-      </slot>
```

Comment:
This seems breaking, since the slot allowed for the icon to fallback to a custom one.

Reply:
True! Fixed